pages/design/rule-design.md

```diff
@@ -39,34 +63,56 @@ A test aspect is a part of a test subject that must be available in order to pro
 
-> _For example:_ A rule for 1.4.1 Use of Color has to make an assumption with CSS-properties are used to make a link visually evident. Typically something like `background`, `border`, `color`, `font`, or `text-decoration`.
+> _For example:_ A rule for 1.4.1 Use of Color has to make an assumption which CSS-properties are used to make a link visually evident. Typically something like `background`, `border`, `color`, `font`, or `text-decoration`.
 
 While most assumptions relate to the rule itself, there are some assumptions that apply at other stages of the evaluation:
 
 - It is assumed that the tested web page is the one that has to conform to WCAG 2.0 and that there is no [conforming alternative version](https://www.w3.org/TR/WCAG21/#dfn-conforming-alternate-version).
 
 - It is assumed that the following technologies are accessibility supported: HTML, CSS, WAI-ARIA, ... (See also WCAG-ACT-RULES-CG's [explanation on Accessibility Support](accessibility-support.html)).
 
-## Applicability
+For more details, see [ACT Rules Format: Assumptions](https://www.w3.org/TR/act-rules-format/#assumptions)
 
-Applicability describes which (elements of) web pages should be tested using the rule. These elements are known as test targets. Applicability must be written in plain language, so that it can be used by QA testers. Applicability must rely on well defined properties of the technologies that are tested. For example, a rule may be applicable to all `video` elements, but it can not be applicable to all `object` elements used to show video, unless the term "video" is further defined.
+## Accessibility Support
 
-Finding objective definitions to use in rules can be difficult, if not outright impossible in some cases. The intent here is to ensure repeatability of the rule. Not everything in WCAG testing is entirely repeatable, but when it comes to rule applicability, this is a hard requirement.
+Information on any known support issues for a feature relevant to the rule.
 
-For more details, see [ACT Rules Format: Applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability)
+For more details, see [ACT Rules Format: Accessibility Support](https://www.w3.org/TR/act-rules-format/#accessibility-support)
 
-## Expectations
+## Background
 
-The applicability help the testers (or test tools) identify what has to be checked. Following that, the expectations are statements that must be true for the applicable elements to pass the rule. If any of the expectations is false, than the target element failed the rule.
+Optional links to information that is relevant reading, such as a success criterion, understanding or technique document, or other specification. 
```

Comment:
It seems that we are moving toward not that "optional" background, and more structured than now.
Notably, WCAG SC and techniques should be here if used in the rule, grouped by category (there was a PR about updating the background in the templates, but I can't find it anymore…)
I also feel that any external document cited in the rule should appear in the background (#930), which could be mentioned here. But maybe we need to agree on it first 😄

Reply:
At the moment the ACT Rules format describes "Background" as an optional section. I agree it is rarely omitted in our rules, but also agree that it should be agreed elsewhere before changing that in this page.